src/Servers/Kestrel/samples/Http3SampleApp/Http3SampleApp.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <!-- Turn off warnings about using preview features -->
```

Comment:
Nit: might be good to add something like "because HTTP/3 is currently in preview state" or something here, so the intent is clear to someone in the future.